api/routes.py

```diff
@@ -826,10 +826,9 @@ def odl_notify(loan_id):
 
 
 # Controllers used for operations purposes
-@app.route("/heartbeat")
```

Comment:
Should we leave the `/heartbeat` endpoint in for a deployment cycle or two. The [load balancers use it for their health check](https://github.com/ThePalaceProject/hosting-playbook/blob/e43aee64deab283c17a52d941a6b9667d8f6e9ba/terraform/modules/web/main.tf#L61-L65). Removing it now would mean that deployment would need to happen in lockstep with this release, since the CM would treat "heartbeat" as a library short name and return a 404.